velox/dwio/parquet/reader/TimestampColumnReader.h

```diff
@@ -70,7 +76,7 @@ class TimestampColumnReader : public IntegerColumnReader {
       const RowSet& rows,
       const uint64_t* /*incomingNulls*/) override {
     auto& data = formatData_->as<ParquetData>();
-    // Use int128_t as a workaroud. Timestamp in Velox is of 16-byte length.
+    // Use int128_t as a workaround. Timestamp in Velox is of 16-byte length.
     prepareRead<int128_t>(offset, rows, nullptr);
     readCommon<IntegerColumnReader, true>(rows);
```

Comment:
I think the clean way to do it is to create our own `ColumnVisitor` instance with a subclass of `TimestampRange` that is dedicated for Parquet Int96, instead of reusing the `readCommon` for integers.  Almost all of the filter type and `valueSize_` dispatches on integers do not apply to timestamp type anyway.

Reply:
I will take further look. Thanks for the review.

Reply:
@Yuhta Updated following above suggestion. A range dedicated for Parquet int96 is added and will replace the original TimestampRange. Would you like to review again? Thanks.